pjmedia/src/pjmedia/conference.c

```diff
@@ -1278,6 +1278,12 @@ PJ_DEF(pj_status_t) pjmedia_conf_get_port_info( pjmedia_conf *conf,
 
     info->slot = slot;
     info->name = conf_port->name;
+    if (conf_port->port) {
+        pjmedia_format_copy(&info->format, &conf_port->port->info.fmt);
+    } else {
+        pj_bzero(&info->format, sizeof(info->format));
+        info->format.id = (pj_uint32_t)-1;
```

Comment:
Do we need to introduce a new format in format.h, such as
```
typedef enum pjmedia_format_id
{
...
PJMEDIA_FORMAT_INVALID = 0xFFFFFFFF
```

Reply:
agree, sounds better than -1 constant.